core/templates/dev/head/pages/exploration_editor/switch_editor_directive.html

```diff
@@ -0,0 +1,38 @@
+<script type="text/ng-template" id="inline/switch_editor_directive">
+  <div class="switch-editor-btn-container">
+    <button type="button" class="btn btn-default oppia-switch-editor-btn" ng-click="setEditorModeToFull()" ng-if="isEditorInSimpleMode()">
+      Use Advanced Editor
+    </button>
+    <button type="button" class="btn btn-default oppia-switch-editor-btn" ng-click="setEditorModeToSimple()" ng-if="isEditorInFullMode()">
+      Use Simple Editor
+    </button>
+  </div>
+  <style>
+  .switch-editor-btn-container {
+    position: fixed;
+    bottom: 0px;
```

Comment:
Alphabetize please -- bottom, then position, then right, then width. Also prefer 0 to 0px (here and elsewhere).